CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.6.2
```

Comment:
@syjust-alma Yes it is because version 2.6.3 was deployed from version 2.6.1 and not merger on master unlike version 2.6.2.
But I will make sure to put the right version once the release is prepared.